Tags rework, highlight old CI runs and DNM

fabiobaltieri opened the pull request:
Rework the tag visualization, show old CI runs and DNM PRs (which should not be there).

Looks somewhat like this:

![x](https://github.com/user-attachments/assets/973e5fc5-5f18-43d2-8206-4bfdec57f0f3)

![x](https://github.com/user-attachments/assets/4e00e9c8-a8b5-4cb1-ac17-4b3f466f4d60)

fabiobaltieri committed:
Flag PR with old CI

Add a feature to flag pull requests that have a CI run older than 31
days. Just add the internal data, no UI changes just yet.

Signed-off-by: Fabio Baltieri <fabiobaltieri@google.com>

fabiobaltieri committed:
Drop two leftover debug prints

These were not meant to be merged in the first place.

Signed-off-by: Fabio Baltieri <fabiobaltieri@google.com>

kartben commented:
I've mentioned it in Zoom but you've probably missed it -- I think relying on color, and even more so where the contrast is so low between all colors, is not good (or at least not enough).

Can you please look at adding something like an 🅾️ ("old"/"outdated"), or "👨🏽‍🦳" or something along these lines (🔄?) either in the PR title, or as a new column?

fabiobaltieri commented:
> Can you please look at adding something like an 🅾️ ("old"/"outdated"), or "👨🏽‍🦳" or something along these lines (🔄?) either in the PR title, or as a new column?

Yeah that's a good idea, last column is already "labels", can do a "flags" column and have stuff like hotfix, trivial, dnm, old ci etc... more flexible than colors too and does not mess the CSS

fabiobaltieri committed:
Rework the tags visualization, add old ci

Rework the tag visualization, move the column just after the title, make
them stand out properly, add old ci as well.

Signed-off-by: Fabio Baltieri <fabiobaltieri@google.com>

fabiobaltieri committed:
Add DNM label highlighting

Look for for DNM labels in the dashboard and put a well visible tag on
it. This should never happen but github is weird these days so at least
show it clearly here. Not skipping these since if this condition happens
it means that CI passed (which it shouldn't) and we need to restart it
or open/close the PR or something.

Signed-off-by: Fabio Baltieri <fabiobaltieri@google.com>

fabiobaltieri commented:
@kartben alright reworked the whole tag story
